src/components/Searchbar/Searchbar.js

```diff
@@ -0,0 +1,115 @@
+import { gray } from "d3-color";
+import { useRouter } from "next/router";
+import React, { useEffect, useState } from "react";
+import AsyncSelect from "react-select/async";
+
+const searchBarStyles = {
+  control: (provided) => ({
+    ...provided,
+    fontFamily: "Gotham-Book",
+    color: "white",
+    backgroundColor: gray,
+    fontSize: 16,
+    border: "0px",
+    borderRadius: "8px",
+    padding: "4px",
+  }),
+  option: (provided, state) => ({
+    ...provided,
+    fontFamily: "Gotham-Book",
+    fontSize: 14,
+    color: "black",
+  }),
+  menu: (styles) => ({
+    ...styles,
+    border: "0px",
+    borderRadius: "8px",
+  }),
+  input: (provided) => ({
+    ...provided,
+    color: "white",
+  }),
+  placeholder: (provided, { isFocused }) => ({
+    ...provided,
+    color: isFocused ? "gray" : "white",
+  }),
+};
+
+//search component for search page
+function Searchbar() {
```

Comment:
```suggestion
const Searchbar = () => {
```

use a arrow function here for consistency